src/main/scala/amba/axi4/AsyncCrossing.scala

```diff
@@ -86,7 +96,7 @@ class AXI4AsyncCrossing(params: AsyncQueueParams = AsyncQueueParams())(implicit
   }
 }
 
-/** Synthesizeable unit tests */
+/** Synthesizable unit tests */
```

Comment:
actually this is not a Scala doc.
```suggestion
// Synthesizable unit tests
```